tests/Options/CiRunOptionsTest.php

```diff
@@ -106,25 +106,26 @@ public function testMissingRequiredOptions($options): void {
     $this->createCiRunOptions($options);
   }
 
-  public function providerMissingRequiredOptions(): array {
+  public static function providerMissingRequiredOptions(): array {
+    $obj = new CiRunOptionsTest("testMissingRequiredOptions");
     return [
       'No options' => [[]],
       'Missing job' => [
         [
-          'phase' => $this->validPhaseName(),
-          'sut' => $this->validSutName(),
+          'phase' => $obj->validPhaseName(),
+          'sut' => $obj->validSutName(),
```

Comment:
I would just make all these helper methods static on `CiJobTestBase` instead of duplicating their logic.

Reply:
We are using all this helper functions from the `CiEnumsTestTrait`, do you mean we need to make them `static` there?

Reply:
Yes, let's make them static anywhere they appear.